docs/4.0/utilities/position.md

```diff
@@ -38,7 +38,7 @@ Position an element at the bottom of the viewport, from edge to edge. Be sure yo
 
 Position an element at the top of the viewport, from edge to edge, but only after you scroll past it. The `.sticky-top` utility uses CSS's `position: sticky`, which isn't fully supported in all browsers.
 
-**Microsoft Edge and IE11 will render `position: sticky` as `position: relative`.** As such, we wrap the styles in a `@supports` query, limiting the stickiness to only browsers that properly can render it.
+**IE11 and IE10 will render `position: sticky` as `position: relative`.** As such, we wrap the styles in a `@supports` query, limiting the stickiness to only browsers that properly can render it.
```

Comment:
"that properly can render it" > "that can render it properly" ?

Reply:
Agreed that "that can render it properly" would read better.

Reply:
Works for me! Feel free to (force) push a change for that and we can merge as soon as CI stuff gets sorted.